### DISCUSSION TITLE
Developer console

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#333] [Developer Feedback]
In the developer console in the statistics I would like to see the number of downloads divided by watch models (numeric or percentage data).

I would also like to be able to filter data for the last month/year. This can be done now by setting the start and end date, but it is not very convenient.

**Comment by silver-zepp** [comment]
Really good suggestions. Forwarded your feedback to our cloud team, let's see if we can get any of those features implemented.
